paddle/fluid/pir/transforms/pd_op_to_kernel_pass.cc

```diff
@@ -397,6 +397,26 @@ static pir::Value AddPlaceTransferOp(pir::Value in,
         {"kernel_name", pir::StrAttribute::get(ctx, copy_kernel_name)},
         {"kernel_key", KernelAttribute::get(ctx, copy_kernel_key)},
         {"dst_place_type", pir::Int32Attribute::get(ctx, 0)}};
+  } else if ((src_place.GetType() == phi::AllocationType::CPU) &&
```

Comment:
```suggestion
  } else if ((src_place.GetType() == phi::AllocationType::CPU) &&
```
这里是不是抽出一个函数比较好？后续可能会有其他国产硬件接入，都按之前GPU上的写法不大好维护吧，不如从现在开始改下。

Reply:
改写为lambda